inspektor/lint.py

```diff
@@ -54,9 +54,13 @@ def __init__(self, args, logger=logging.getLogger('')):
         self.log = logger
         self.args = args
         self.verbose = args.verbose
+        if hasattr(args, 'parallel'):
```

Comment:
I was really intrigued on why this was necessary: basically the `default` should set this always.  Then I remember that sometimes lint is called by `checkall`, so yes, LGTM.

Reply:
exactly (and Travis actually reminded me this on this very PR)